### PR TITLE
script to determin if sentence is a question

### LINKED ISSUE #53: Create Python algorithm to determine question and sentiment
Use the NLTK to determine the following for a sentence:
- Is it a question or just a statement (or garbled text)
- What is the sentiment of the sentence - positive, negative, neutral

Keep in mind that feedback can be more than one sentence.

### PR DESCRIPTION
**Pull request by Abenezer1041**
Close #53 

**Task Description**

This code is uses nltk library to determining if a sentence is a question or statement.

**Time Spent**

| **Activity** | **Time Spend** |
| ----------- | --------- |
|Research  | 2 hours |
| writing code and testing | 4 hours |

- Import the following libraries 

      1. import nltk

- To Import, type this commend on the terminal 
      install pip3 nltk

      

**Instruction**

-  Run the code
- Enter a sentence .
- You should get result based on the sentence you entered. 

**Result**
Screen Shot 02

### CODE REVIEW

**Review by eyerusalemdani — APPROVED** [review]
I have tried some statements as input for test, it works well
Abe PR review
.

**Comment by dale108** [comment]
What if we have a statement like "you shouldn't do that."? won't this script think it's a question because it has the word "shouldn't" in it?

**Comment by Abenezer1041** [comment]
> What if we have a statement like "you shouldn't do that."? won't this script think it's a question because it has the word "shouldn't" in it?

No it should not because its only checking for the first word.

**Review by dale108 — APPROVED** [review]

File: backend/question.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    
+    for pattern in Question_Words:
+        if pattern in sentence.split()[0] or  type=="whQuestion" or type== "ynQuestion"  :
+            print("This is a question")
+            exit()                      
+             
+    else:        
+        print("This is a statement") 
+         
+question="  are you smart"
```
[the remark this answers is not in the document]
Comment (Abenezer1041) [reply backend/question.py]: i removed it

**Pull request merged**